### ISSUE TITLE
': void' can be declared as return type hint false positive

### ISSUE THREAD

**Issue by MMauro94** [kalessil/phpinspectionsea#861]
I found a false-positive on the mentioned inspection. Here's some test functions to reproduce it:
```
public static function f1($param = null) {
	return $param;
}
```

This happens specifically when there is a not type-hinted param with a `null` default value.

**Comment by funivan** [comment]
Confirmed. This is false positive

**Comment by kalessil** [comment]
Fixed
